docs/api.rst

```diff
@@ -12661,3 +12661,39 @@ Get all valid locations
 
   :statuscode 200: Information was correctly returned
   :statuscode 500: Internal rotki error
+
+Refetch general cache
```

Comment:
Both here in the docs and everywhere else in docs and in the code please use the term "Refresh" instead of "Refetch"